.mdb

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I am not familiar with mdb server. Can you elaborate on that? Also, for HTML or javascript code there are many other options to run it, like https://codepen.io 
That said, you want to write instructions on how to use template. Demo is not required but a good README text to get started is more important.